partner_communication_switzerland/__manifest__.py

```diff
@@ -30,14 +30,15 @@
 # pylint: disable=C8101
 {
     'name': 'Compassion CH Partner Communications',
-    'version': '10.0.2.4.2',
+    'version': '10.0.2.4.3',
     'category': 'Other',
     'author': 'Compassion CH',
     'license': 'AGPL-3',
     'website': 'http://www.compassion.ch',
     'depends': [
         'report_compassion',
         'child_switzerland',
+        'partner_compassion'
```

Comment:
This is useless because report_compassion depends on sponsorship_switzerland and sponsorship_switzerland already depends on partner_compassion. You can remove it.